_data/accessibility-tests/file-input.yml

```diff
@@ -41,63 +47,57 @@ test_items:
     github_issue_number: 5616
     github_issue_repo: uswds
     test_type: keyboard
-    version_tested: 3.8.1
+    version_tested: 3.8.0
     wcag_criterion: 2.4.7
   - summary: File can be input with only a keyboard.
     summary_additional: You can successfully upload a file using only a keyboard. No mouse or dragging action is required.
     test_status: pass
     test_type: keyboard
-    version_tested: 3.8.1
+    version_tested: 3.8.0
     wcag_criterion: 2.5.7
   - summary: Content doesn't change until the user takes an action to change it.
     summary_additional: When you access the file input with a keyboard, content stays the same until you use keys to make changes.
     test_status: pass
     test_type: keyboard
-    version_tested: 3.8.1
+    version_tested: 3.8.0
     wcag_criterion: 3.2.2
 # Screen reader tests
   - summary: Screen reader announces file input.
     summary_additional: When you navigate with arrow keys and the tab key while using a screen reader, you hear the prompt to "drag file here or choose from folder" and you hear any related hints or instructions.
-    test_status: pass
+    test_status: exception
+    status_details: N/A
+    github_issue_number: N/A
+    github_issue_repo: N/A
```

Comment:
@amyleadem @amycole501 @alex-hull @finekatie Is there github issue info for this now that we've changed it to `exception`. Also, how would we like to phrase the `status_details` and update the `summary_additional`?

Reply:
I defer to @amycole501 and @alex-hull about whether there's a GH issue. I don't know quite what's happening with this - notes in our spreadsheet appear older to me and it still reads as a passed test.

Reply:
Resolved in [2a91b16](https://github.com/uswds/uswds-site/pull/2963/commits/2a91b1666fa6ea76c0d637a80fed290d33e9ed7b)